operator/v1alpha1/server.proto

```diff
@@ -21,8 +57,25 @@ message GetMetricsRequest {
     repeated LabelSelector conditions = 5;
 }
 
-message GetMetricsResponse {
-    repeated Result results = 1;
+message GetMetricResponse {
+    MetricResult result = 1;
```

Comment:
What if GetMetricRequest() queries contains multiple targets (selected by the LabelSelector argument) ?
Do we need a "repeated" decorator here?

Reply:
@mamafun 
Yes, please review the commitment, it has been revised.